openmetadata-ui/src/main/resources/ui/src/components/ActivityFeed/ActivityThreadPanel/AnnouncementThreads.tsx

```diff
@@ -166,10 +167,15 @@ const AnnouncementThreads: FC<ActivityThreadListProp> = ({
   return (
     <div className={className}>
       {getAnnouncements(activeAnnouncements)}
-      <Typography.Text className="tw-block tw-mt-4 tw-font-medium">
-        Inactive Announcements
-      </Typography.Text>
-      <Divider className="tw-mb-4 tw-mt-2" />
+      {!isEmpty(inActiveAnnouncements) && (
```

Comment:
```suggestion
      {inActiveAnnouncements.length && (
```

Let's avoid using negative condition to improve code readability

Reply:
Done.